packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -612,6 +612,38 @@ export namespace EmrCreateCluster {
     readonly weightedCapacity?: number;
   }
 
+  /**
+   * On-Demand Allocation Strategies
+   *
+   * Specifies the strategy to use in launching On-Demand instance fleets. Currently, the only option is "lowest-price" (the default), which launches the lowest price first.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-emr-instancefleetconfig-ondemandprovisioningspecification.html
+   *
+   */
+  export enum OnDemandAllocationStrategy {
+    /**
+     * Lowest-price, which launches instances from the lowest priced pool that has available capacity.
+     */
+    LOWEST_PRICE = 'lowest-price',
+  }
+
+  /**
+   * The launch specification for On-Demand Instances in the instance fleet, which determines the allocation strategy.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-emr-instancefleetconfig-ondemandprovisioningspecification.html
+   *
+   */
+  export interface OnDemandProvisioningSpecificationProperty {
```

Comment:
@lpizzinidev 

I originally added it because it was in the API reference, but the CloudFormation document apparently does not support it, so I deleted it.

Is this necessary?
In my opinion, it should follow CloudFormation, so I didn't think it was necessary to add it.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-emr-instancefleetconfig-ondemandprovisioningspecification.html

Reply:
I added them. 

https://github.com/aws/aws-cdk/pull/27791/commits/6510ab0af9360ba7b47c38832cb367a7f1c810f5

Reply:
You're right, CFN is the right reference in this case. 
Thanks for clarifying.